initials.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+
+int main()
+{
+	for(int i = 0; i<2;i++)
+	{
+		printf("#####         ##\n");
+		printf("##            ##\n");
+
+	}
+
+	printf("#####         ########\n");
+
```

Comment:
you forgot return 0; and to put any comments. Make sure to follow guidelines - they're important, especially when you get to coding more complex programs.